node/child_process_test.ts

```diff
@@ -67,7 +67,7 @@ Deno.test({
       });
       childProcess.stdin.write("  console.log('hello')", "utf-8");
       childProcess.stdin.end();
```

Comment:
When the `'exit'` event is fired, stdin and stdout have not yet been closed and consumed. The `'close'` event should have been used here.